src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs

```diff
@@ -100,8 +100,8 @@
       {:path (str "/workspaces/" (ws-path workspace-id) "/checkBucketReadAccess")
        :method :get})
 
-(defn update-workspace-acl [workspace-id]
-  {:path (str "/workspaces/" (ws-path workspace-id) "/acl")
+(defn update-workspace-acl [workspace-id invite-new?]
+  {:path (str "/workspaces/" (ws-path workspace-id) "/acl?inviteUsersNotFound=" (some? invite-new?))
```

Comment:
After poking around a bit, since I'm working on the only place where this code is called, I've decided to just use `true?` and leave it at that.